app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -197,6 +247,28 @@ class HomePresenter(
         }
     }
 
+    /**
+     * Handles buy token for fiat deeplink
+     */
+    private fun handleBuyDeeplink(data: DeeplinkData) {
```

Comment:
I think this logic can be extracted to HomePresenterDeeplinkHandler, for example